src/app.tsx

```diff
@@ -9,13 +9,16 @@ import { UserContext } from './common/user-context';
 const App = () => {
   const [filteredUsers, setFilteredUsers] = useState<DummyUser[]>([]);
   const [unfilteredUsers, setUnfilteredUsers] = useState<DummyUser[]>([]);
+  const [currentUser, setCurrentUser] = useState<LoginResponse>({} as LoginResponse);
```

Comment:
Whenever possible try to not assert types, since it can lead to errors that typescript would prevent. I'd recommend having `const [currentUser, setCurrentUser] = useState<LoginResponse | null>(null);` and checking for `null` where necessary